rules-tattletale/addon/src/main/java/org/jboss/windup/rules/apps/tattletale/TattletaleRuleProvider.java

```diff
@@ -145,5 +147,18 @@ private void createReportModel(GraphContext context, FileModel input, String rep
             ReportService reportService = new ReportService(context);
             reportService.setUniqueFilename(applicationReportModel, "tattletale" + "_" + inputProjectModel.getName(), "html");
         }
+
+        private boolean isTattleTaleReportGenerationEnabled(GraphRewrite event)
+        {
+            Boolean enableReport = (Boolean) event.getGraphContext().getOptionMap().get(EnableTattletaleReportOption.NAME);
+            Boolean disableReport = (Boolean) event.getGraphContext().getOptionMap().get(DisableTattletaleReportOption.NAME);
+            Collection<String> targets = (Collection<String>) event.getGraphContext().getOptionMap().get(TargetOption.NAME);
+            boolean eapTarget = targets != null && targets.stream().anyMatch(target -> target.startsWith("eap"));
+            if (eapTarget && BooleanUtils.isTrue(disableReport) && BooleanUtils.isNotTrue(enableReport))
+                return false;
+            else if (!eapTarget && BooleanUtils.isNotTrue(enableReport))
+                return false;
+            return true;
```

Comment:
At this point what I see is that we cover this cases :
1. eap = true , disable = true, enable = true ----> false
2. eap = false, enable = false ---> false

But we are covering these 2 cases as true :
4. eap = false, enable = true ---> true
5. eap = true, disable = false, enable = false ---> true
Are we totally sure with case #5 ?

Reply:
Just to keep trace, we talked about the use cases in IRC and figured out that these options are `true` if provided in command line and `false` otherwise.

Reply:
+1